web/locales/en.json

```diff
@@ -160,7 +160,7 @@
         "emissionsOfStored-question": "What about emissions from generating electricity which is then stored in batteries or used to fill reservoirs?",
         "emissionsOfStored-answer": "As currently only a small proportion of electricity is stored, inaccuracies in the modelling of these emissions should not make much difference to the total at present. However given the increasing importance of storage we hope to take it into account more fully in our model shortly.",
         "guaranteesOfOrigin-question": "What are green certificates, and how are they taken into account?",
-        "guaranteesOfOrigin-answer": "When producers of renewable energy produce electricity, they can create Guarantees of Origin (or Renewable Energy Certificates) - a proof that renewable electricity has been produced and distributed into the power grid. These guarantees can then be sold, giving other non-renewable electricity producers a way to \"compensate\" for their emissions and the right to claim that the electricity they produce comes from renewable sources, regardless of its actual, physical source.<br><br> This means that electricity consumers can be told that their electricity is green, when this is not true in the physical sense. This problematic, as it removes consumer incentives to consume electricity at the best time (for instance when the amount of renewable electricity in the grid is at its highest). This map therefore excludes Guarantees of Origin, instead offering a location-based, physical picture of the power grid, in order to responsibilize consumers.",
+        "guaranteesOfOrigin-answer": "When producers of renewable energy produce electricity, they can create Guarantees of Origin (or Renewable Energy Certificates) - a proof that renewable electricity has been produced and distributed into the power grid. These guarantees can then be sold, giving electricity retailers a way to claim that the electricity they resell comes from renewable sources, regardless of its actual, physical source.<br><br> This means that electricity consumers can be told that their electricity comes from solar and wind, even in a windless night. This problematic, as it removes consumer incentives to consume electricity at the best time (for instance when the amount of renewable electricity in the grid is actually at its highest). This map therefore excludes Guarantees of Origin, instead offering a location-based, physical picture of the power grid, in order to responsibilize consumers.",
```

Comment:
'to responsibilize' is kind of a weird choice of phrasing (not technically incorrect though), it doesn't sound right to an english speaker. How about 'to empower' instead?